src/main/resources/assets/forge/lang/en_US.lang

```diff
@@ -140,8 +140,56 @@ fml.configgui.tooltip.undoChanges=Undo Changes
 fml.configgui.tooltip.default=[default: %s]
 fml.configgui.tooltip.defaultNumeric=[range: %s ~ %s, default: %s]
 
+fml.error.backupLevelIsBeingUsed=Forge Mod Loader detected that the backup level.dat is being used.\n\nThis may happen due to a bug or corruption, continuing can damage\nyour world beyond repair or lose data / progress.\n\nIt's recommended to create a world backup before continuing.
+fml.error.common.fileContainsMoreInfo=The file 'logs/fml-client-latest.log' contains more information
+fml.error.common.modsCouldNotBeFound=The mods and versions listed below could not be found
+fml.error.common.problemWithMinecraftInstallation=Forge Mod Loader has found a problem with your Minecraft installation
+fml.error.common.couldNotConnectToServer=Forge Mod Loader could not connect to this server
+fml.error.duplicatedModSources=You have mod sources that are duplicate within your system
+fml.error.errorSavingTheArchive=There was an error saving the archive %s
+fml.error.fixTheProblemAndTryAgain=Please fix the problem and try again
+fml.error.forbiddenModdedAccess=The server %s has forbidden modded access
+fml.error.minimumRequiredVersion=%s : minimum version required is %s
+fml.error.modIdFileName=Mod Id : File name
+fml.error.oldSaveLoadConfirm.1=The world %s contains pre-update modding data
+fml.error.oldSaveLoadConfirm.2=There may be problems updating it to this version
+fml.error.oldSaveLoadConfirm.3=FML will save a zip to %s
+fml.error.oldSaveLoadConfirm.4=Do you wish to continue loading?
+fml.error.sortingProblem.1=A mod sorting cycle was detected and loading cannot continue
+fml.error.sortingProblem.2=The first mod in the cycle is %s
+fml.error.sortingProblem.3=The remainder of the cycle involves these mods
+fml.error.sortingProblem.4=%s : before: %s, after: %s
+fml.error.theyAreRequiredToPlayOnServer=They are required to play on this server
+fml.error.wrongMinecraftVersion.1=The mod listed below does not want to run in Minecraft version %s
+fml.error.wrongMinecraftVersion.2=%s (%s) wants Minecraft %s
+
+fml.menu.config=Config
+fml.menu.disable=Disable
+fml.menu.disabled=DISABLED
+fml.menu.metaNotFound=Metadata not found
+fml.menu.modinfo.versionFormatted=Version: %s (%s)
+fml.menu.modinfo.modIdAndState=Mod ID: '%s' Mod State: %s
+fml.menu.modinfo.credits=Credits:
+fml.menu.modinfo.authors=Authors:
+fml.menu.modinfo.url=URL:
+fml.menu.modinfo.noChildMods=No child mods for this mod
+fml.menu.modinfo.childMods=Child mods:
+fml.menu.modinfo.updateAvailable=Update Available:
+fml.menu.modinfo.version=Version:
+fml.menu.modinfo.modState=Mod State:
+fml.menu.modinfo.noInformation=No mod information found
+fml.menu.modinfo.nomcmodFile=Ask your mod author to provide a mod mcmod.info file
+fml.menu.modinfo.changes=Changes:
+fml.menu.modlist=Mod List
+fml.menu.modoptions=Mod Options...
 fml.menu.mods=Mods
+fml.menu.modsLoadedActiveSingular=%d mod%s loaded, %d mod%s active
+fml.menu.modsLoadedPlural=s
```

Comment:
Not all languages operate like this, entire string should be translated for singular/plural.